### ISSUE TITLE
Release v1.57.6

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#7555]
:robot: A dedicated issue to run tests and deploy release [v1.57.6](https://github.com/deckhouse/deckhouse/milestone/286).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.57.6 | test-v1.57.6*` or a branch `main | release-1.57`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.25 | 1.26 | 1.27 | 1.28 | 1.29`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.57.6 | test-v1.57.6*` or a branch `main | release-1.57`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.57
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [60aa64](https://github.com/deckhouse/deckhouse/commit/60aa64d8db61255fdb63c512db9d0a060f58d8e4) in branch **main**:
```
Re-generate changelog v1.57.6 (#7556)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: deckhouse-BOaTswain <deckhouse-BOaTswain@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c70f12](https://github.com/deckhouse/deckhouse/commit/c70f12edaaa59ce3f7a4f69ab9481530d2a904bd) in branch **main**:
```
Update action.yml to include a new directory in the module path. (#7561)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6f17c5](https://github.com/deckhouse/deckhouse/commit/6f17c58a44325b1e4eec4f752275c4cb756056ba) in branch **main**:
```
[shell_lib] Fix an error with password generation in shell hooks (#7548)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8f4a01](https://github.com/deckhouse/deckhouse/commit/8f4a015d6c5fa69252e1cba2bd6d66c98661d626) in branch **main**:
```
[upmeter] tlsConfig (#7495)

Signed-off-by: Ilya Sosnovsky <ilya.sosnovsky@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4be786](https://github.com/deckhouse/deckhouse/commit/4be786e83c6b37cfbdfad676f550a45ea6954163) in branch **main**:
```
[node-local-dns] Fix node local dns readiness probes (#7553)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```
